frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -54,47 +163,211 @@
+      selectedDateFilterType: null,
+      selectedPaymentFilterTypes: [],
+      startDateFrom: null,
+      startDateTo: null,
+      menuStartDateFrom: false,
+      menuStartDateTo: false,
       headers: [
-        { title: 'FA Number', key: 'faNumber' },
-        { title: 'FA Type', key: 'faType' },
-        { title: 'Facility', key: 'facility' },
+        { title: 'Funding Agreement Number', key: 'fundingAgreementNumber' },
+        { title: 'Funding Agreement Type', key: 'fundingAgreementType' },
+        { title: 'Facility Name', key: 'facility' },
+        { title: 'Signing Expense Authority', key: 'expenseAuthorityName' },
         { title: 'Start Date', key: 'startDate' },
         { title: 'End Date', key: 'endDate' },
         { title: 'Status', key: 'status' },
         { title: 'Actions', key: 'actions' },
       ],
+      dateFilterTypes: [
+        { label: '3 Months', value: '3Months' },
+        { label: '6 Months', value: '6Months' },
+        { label: 'YTD', value: 'YTD' },
+        { label: 'Custom', value: 'Custom' },
+      ],
+      paymentFilterTypes: [
+        { label: 'Base Funding', value: 'baseFunding' },
+        { label: 'Supplementary Allowances', value: 'supplementaryAllowances' },
+        { label: 'Other', value: 'other' },
+      ],
+    }
+  },
+  computed: {
+    ...mapState(useAuthStore, ['userInfo']),
+    formattedStartDateFrom() {
+      return this.startDateFrom ? format.formatDate(this.startDateFrom) : ''
+    },
+    formattedStartDateTo() {
+      return this.startDateTo ? format.formatDate(this.startDateTo) : ''
     }
   },
+  async created() {
+    this.FUNDING_AGREEMENT_STATUS_CODES = FUNDING_AGREEMENT_STATUS_CODES
+    this.initialize()
+    await this.getOrgUsers() // Retrieve the list of users to retrieve the expense authority name
+    await this.getFundingAgreements()
+  },
+  methods: {
+    initialize() {
+      this.selectedFacility = null
+      this.selectedDateFilterType = '3Months'
+      this.selectedPaymentFilterTypes = ['baseFunding', 'supplementaryAllowances', 'other']
+      this.startDateFrom = null
+      this.startDateTo = null
+    },
+
+    async getOrgUsers() {
+      try {
+        this.loading = true
+        const res = await ApiService.apiAxios.get(ApiRoutes.USER_PERMISSIONS_FACILITIES + '/' + this.userInfo.organizationId)
+        this.orgUsers = res.data
+      } catch (error) {
+        this.setFailureAlert('Failed to get the list of users by organization id: ' + this.userInfo.organizationId, error)
+      } finally {
+        this.loading = false
+      }
+    },
+
+    /** 
+     * TODO: Payment type is displayed in UI but is not yet integrated into this method as required CRM data is not yet in place.
+     * This is a know issue and postponed to a future sprint.
+     */
+    async getFundingAgreements() {
+      try {
+        this.loading = true
+        this.fundingAgreements = []
+        const startDateThreshold = this.startDateThreshold()
```

Comment:
There's no benefit to defining a variable here as it's only used once and computed() methods should just be treated as properties.

Reply:
Fixed.